javascript-loops/main.js

```diff
@@ -0,0 +1,65 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
```

Comment:
Hey Sean, you are creating 2 counter variables here : 'i' and 'currentNumber'. Please only use currentNumber to iterate through the loop.